### PR TITLE
create address form item and add events for file upload

### PR DESCRIPTION
**Pull request by MohamedTaha880**

### CODE REVIEW

File: src/modules/form/assets/js/form.js [hunk 1]
```diff
@@ -210,6 +211,109 @@ $(document).ready(function() {
   });
  }
 
+ function InitializeAdressInputs(){
```
Comment (Aldus83) [inline src/modules/form/assets/js/form.js]: @MohamedTaha880 typo Adress -> address

File: src/modules/form/assets/js/form.js [hunk 2]
```diff
@@ -210,6 +211,109 @@ $(document).ready(function() {
...
+  $('.address-input').each(function() {
+    const input = $(this);
+    const loader = $('#loader'); // Reference to the loader element
+    // Create a hidden input to store the full address
+    const roadInput = $('input[name="road"]');
```
Comment (Aldus83) [inline src/modules/form/assets/js/form.js:219]: @MohamedTaha880  what is the use of constant here? 
Why can't you use normal variables, and directly assign them?

**Review by Aldus83 — CHANGES_REQUESTED** [review]

File: src/modules/form/assets/js/form.js (hunk 1 above)
Comment (MohamedTaha880) [reply src/modules/form/assets/js/form.js]: Done

File: src/modules/form/assets/js/form.js (hunk 2 above)
Comment (MohamedTaha880) [reply src/modules/form/assets/js/form.js:219]: @Aldus83 
Sorry I changed the approach but I forgot this comment
I updated it
We can add any inputs of them in our form 
For example:
[INPUT|required|class=d-none|type=hidden|id=state|name=state|value=[J|name=state|no_qtags]:shadow-form]
[INPUT|required|class=d-none|type=hidden|id=postcode|name=postcode|value=[J|name=postcode|no_qtags]:shadow-form]

**Pull request merged**